en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -17,115 +17,114 @@ review-ticket: https://github.com/programminghistorian/ph-submissions/issues/374
+### Downloading Comments and Metadata
 
-## Downloading Comments and Metadata
-Equipped with the video IDs for the videos you selected in the [Video Selection phase](#Video-Selection), navigate to the [Video Comments tab](https://ytdt.digitalmethods.net/mod_video_comments.php) on the YouTube Data Tools site. Enter the first video ID in the "Video id:" field. 
+Equipped with the video IDs for the videos you selected in the [Video Selection phase](#Video-Selection), navigate to the [Video Comments tab](https://ytdt.digitalmethods.net/mod_video_comments.php) on the YouTube Data Tools site. Enter the first video ID in the **Video id:** field. 
 
-For ethical purposes, you may choose to have the tool add irreversible hashes to the comment username and ID numbers by clicking on "Pseudonymize." Leave the default selection for a .csv file output format and select "Submit query." Repeat this process for each video. More details on this process can be found in Bernhard Reider's [instructional video](https://www.youtube.com/watch?v=EnTy_pbkCfM).
+For ethical purposes, you may choose to have the tool add irreversible hashes to the comment username and ID numbers by clicking on _Pseudonymize_. Leave the default file format selection to `.csv`, and press _Submit_. Repeat this process for each video. More details on this process can be found in Bernhard Reider's [instructional video](https://www.youtube.com/watch?v=EnTy_pbkCfM).
 
-You have three choices to make about how your data output will be formatted. First, select if you’d like to generate HTML tables of your results in addition to file exports. This can be useful if [reason why it might be a good option]. Second, select **Pseudonymize** if you’d like to make usernames and comment IDs anonymous.
+For each video ID you enter, YouTube Data Tools will download four files: 
+- The file with the suffix `comments.csv` contains the text of each video comment.
+- The file ending `basicinfo.csv` comprises the video metadata.
+- The files ending in `commentnetwork.gdf` and `authors.csv`, which you don't need for this lesson.
 
-Next navigate to the Video Comments tab on the YouTube Data Tools site. Enter the first video ID in the “Video id:” field.
+Save the four files associated with each video into their own folder, using the video ID as the folder’s name. Next, create a master directory titled `ytdt_data` and save each of these folders inside. You will be using the `ytdt_data` folder in the code below.

```

Comment:
Just slightly confused by the fact we are asked to save the _four_ files, despite the line just above mentioning that two of these files are not needed for the lesson. Perhaps another very short note to clarify here?

Reply:
We could add a paranthentical or following sentence that explains: "You won't need all four files for this lesson, but they contain additional information that might be useful for additional research." We could add more detail or link a reference if you think that would help.